core/capabilities/ccip/oraclecreator/bootstrap.go

```diff
@@ -441,9 +446,15 @@ func (d *peerGroupDialer) closePeerGroup(endpointConfigDigest cciptypes.Bytes32)
 }
 
 func (d *peerGroupDialer) createPeerGroup(
+	ctx context.Context,
 	endpointConfigDigest cciptypes.Bytes32,
 	rmnHomeConfigDigest cciptypes.Bytes32,
 ) error {
+	if ctx.Err() == context.Canceled {
```

Comment:
How about `!= nil` since it could be `DeadlineExceeded`:
```suggestion
	if ctx.Err() != nil {
```